### PR TITLE
Updates to Install and Rules pages 

### PR DESCRIPTION
**Pull request by PrettyCoolPattern**
Compiled from edenxi.com and discord from 5/22/20.

Some formatting changes and info directly added to Install & Rules and some added to config for now

### CODE REVIEW

**Comment by PrettyCoolPattern** [comment]
Ino said I was on an older base so the conflicts should be good in this PR instead

**Comment by PrettyCoolPattern** [comment]
Oh wait, maybe I just am not running prettier on this before committing? @_@;
Ill get this figured out soon I promise lol

**Comment by PrettyCoolPattern** [comment]
Yesss, I did it!!! xD

**Comment by InoUno** [comment]
Hmm, looks like some formatting has been applied to all files and not just JS/TS ones, which is what the `format` command should only be targeting.

Did you just run `npm run format`, or did you also try some other things that might have caused JSON/CSS files to be formatted as well?
We probably want to format those too eventually (better to do in separate PR), I'm just curious if there might be some setup differences that makes the format command hit JSON/CSS files as well for you.

**Comment by PrettyCoolPattern** [comment]
Hmm.. I believe I ran update, upgrade and run-script format as well in this PR, I can rebase and re-pr without running those.

**Comment by InoUno** [comment]
Running `npm update` or `npm upgrade` should only really be done if there's a specific need to update any/all packages. They shouldn't be needed for regular PRs. But it's fine for now, we can keep the updated version in this PR.

Doing those things shouldn't have formatted all the rest of the files though. Gonna try to figure it out with you via discord.

**Comment by InoUno** [comment]
Looks good now regarding the changes (could do without the package updates, but that's fine for now).

I'll let @slassen review the text/content changes.

**Comment by InoUno** [comment]
I took the liberty to restore the package files so they won't cause a lot of conflicts with the other PRs.

**Comment by slassen** [comment]
I like the last updated timestamp on terms and conditions. Maybe we can copy this for each section?

I would rather have some sort of table for the allowed/disallowed plugins instead of the way it's set up in this PR.

I would rather have something in italics instead of another numbered bullet point with a `^` signaling that it relates to the above point.

Please use the previous elements and styling on Discord rules.

For the "Punishments" section please use a semantic table.

**Comment by bkesecker** [comment]
fixes https://github.com/EdenServer/eden-web/issues/5

File: client/src/components/install.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ function install(props) {
   const { info } = props;
 
   return (
-    <Container>
+    <>
```
Comment (bkesecker) [inline client/src/components/install.jsx:11]: Probably invalid XML.  Maybe just comment out?  Can clients override style "display: none"?

Comment (PrettyCoolPattern) [reply client/src/components/install.jsx]: Man.. I had addon interference in my VSC... really messed me up for a minute there lol but then I discovered how to use Prettier, this next PR should fix any formatting issues.

Comment (PrettyCoolPattern) [reply client/src/components/install.jsx]: > Can clients override style "display: none"?

I don't think so, when node.js compiles everything it removes all comment lines in the final client .js build files to make it very efficient for production.

**Comment by bkesecker** [comment]
> ran prettier

Did this change all the double quotes to single quotes?

**Comment by PrettyCoolPattern** [comment]
> > ran prettier
> 
> Did this change all the double quotes to single quotes?

Maybe my prettier config isn't setup right?

**Comment by PrettyCoolPattern** [comment]
> > > ran prettier
> > 
> > 
> > Did this change all the double quotes to single quotes?
> 
> Maybe my prettier config isn't setup right?

Looking into it.

**Comment by bkesecker** [comment]
Looks like the other way around.  Tough to review with so many changes.

**Comment by PrettyCoolPattern** [comment]
image

Hmm.. I guess that it was a default setting, I may have reset some prettier options in VSC ?

I did find a fix for the single quote replace and I will undo that change now.

**Comment by bkesecker** [comment]
> I did find a fix for the single quote replace and I will undo that change now.

Thank you.  Will need to merge with develop to resolve conflicts.

**Comment by PrettyCoolPattern** [comment]
I was able to choose to force either double quote or single quotes with prettier but I couldn't get it to run without interfering..
[pretty sure this article](https://github.com/prettier/prettier-vscode/issues/1085) is whats wrong with end right now, but Ill have to pick back up on it later tonight for now.

**Comment by PrettyCoolPattern** [comment]
I realllly hope I did this right >< I possibly should have changed my merge order slightly but I resolved the conflicts manually which looked like the login-fix pr branch didn't get applied to this branch after some of my reverts and merges, but we should be ready to roll now. Thanks everyone for the patience while I get up and running with git :P

**Comment by slassen** [comment]
No problem. I’m sure you’re almost there! I think it would probably be best if you copied just the files of your changes you wanted to make to your desktop then switch to the develop branch and make a new branch/PR. Additionally, please make sure to review and address the points in my previous comment.

**Comment by PrettyCoolPattern** [comment]
> No problem. I’m sure you’re almost there! I think it would probably be best if you copied just the files of your changes you wanted to make to your desktop then switch to the develop branch and make a new branch/PR. Additionally, please make sure to review and address the points in my previous comment.

I am so into that idea. This one has gotten a bit messy ^^;

**Pull request closed without merging**